### DISCUSSION TITLE
Old Versions

### DISCUSSION THREAD

**Discussion by Nikopoiko123** [Sinytra/ForgifiedFabricAPI#99] [General]
I don't want to sound rude or disrispectful and I'm not programer or enything like that but I love play moded  Minecraft on 1.16.5
if you can make that mod for 1.16.5 I and my friends will be very  greatful. If you can't there there is no problem .
Onece again I dont want sound like annoing customer that thinks he knows best despite don't notknowing anithing about subject.

**Comment by coco875** [comment]
he already answer to this in his [FAQ](https://github.com/Sinytra/Connector/discussions/689) and the answer is no. To explain why the fabric API is already a hudge work and backport on older version it's very complicate between mojang change, fabric api change and forge change you have to remake from scratch at this point and 1.12-1.16-1.18 are deprecanted now by most mod so conflict can appears with other mod and never be fix because the mod authers prefer maintain recent version then old version.
